edx_proctoring/views.py

```diff
@@ -168,6 +170,76 @@ def handle_exception(self, exc):
         return resp
 
 
+class ProctoredExamAttemptView(ProctoredAPIView):
+    """
+    Endpoint for getting timed or proctored exam and its attempt data.
+    /edx_proctoring/v1/proctored_exam/attempt/course_id/{course_id}/content_id/{content_id}
+
+    Supports:
+        HTTP GET:
+            ** Scenarios **
+            ?is_learning_mfe=true
+            returns attempt data with `exam_url_path` built for the learning mfe
+
+            Returns an existing exam (by course_id and content id) with latest related attempt.
+            and active attempt for any exam in progress (for the timer feature)
+            {
+                'exam': {
+                    ...
+                    attempt: { ... }
+                },
+                'active_attempt': { ... },
+            }
+    """
+    def get(self, request, course_id, content_id):
+        """
+        HTTP GET handler. Returns exam with attempt and active attempt
+        """
+        active_attempt_data = {}
+        attempt_data = {}
+        active_exam = {}
+
+        is_learning_mfe = request.GET.get('is_learning_mfe', False)
+        is_learning_mfe = True if is_learning_mfe in ['1', 'true', 'True', True] else False
+
+        active_exams = get_active_exams_for_user(request.user.id)
+        if active_exams:
+            # Even if there is more than one exam, we want the first one.
+            # Normally this should not be an issue as there will be list of one item.
+            active_exam_info = active_exams[0]
+            active_exam = active_exam_info['exam']
+            active_attempt = active_exam_info['attempt']
+            active_attempt_data = get_exam_attempt_data(
+                active_exam.get('id'),
+                active_attempt.get('id'),
+                is_learning_mfe=is_learning_mfe
+            )
+        print('real exam content_id ', active_exam.get('content_id'))
+        if active_exam and active_exam.get('course_id') == course_id and active_exam.get('content_id') == content_id:
+            exam = active_exam
+            exam.update({'attempt': active_attempt_data})
```

Comment:
Do we need to add an attempt into the exam dict since it is the same as the `active_attempt_data` field in the response_dict?